src/aws/cloudfront.tf

```diff
@@ -39,6 +39,7 @@ resource "aws_cloudfront_distribution" "marsha_cloudfront_distribution" {
     allowed_methods  = ["GET", "HEAD", "OPTIONS"]
     cached_methods   = ["GET", "HEAD", "OPTIONS"]
     target_origin_id = "${local.s3_destination_origin_id}"
+    trusted_signers  = ["${var.cloudfront_trusted_signer_id}"]
```

Comment:
missing a "s" to "need" and "url" in the commit message: `if a rule needs to disable signed urls`

Reply:
Thanks